cli.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Sets up the CLI command early in the WordPress load process.
+ *
+ * This is necessary to setup the environment to perform runtime checks.
+ *
+ * @package plugin-check
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'WP_CLI' ) || ! WP_CLI ) {
+	return;
+}
+
+// Check if the plugin autoloading is set up.
+if ( ! class_exists( 'WordPress\Plugin_Check\CLI\Plugin_Check_Command' ) ) {
+	// Check the autoload file exists.
+	if ( ! file_exists( __DIR__ . '/vendor/autoload.php' ) ) {
+		WP_CLI::error( 'Plugin Check autoloaded not found.' );
```

Comment:
```suggestion
		WP_CLI::error(
			__( 'Plugin Check autoloaded not found.', 'plugin-check' )
		);
```

Reply:
@jjgrainger @mukeshpanchal27 Please also fix this to say "Plugin Check autoloader ..." instead of "Plugin Check autoloaded ..."

Reply:
@mukeshpanchal27 @felixarntz I will update to `autoloader` here, but see my other comment on how WordPress is not loaded yet so the translation functions are not available yet and can't be used. https://github.com/10up/plugin-check/pull/100#discussion_r1146027688

Let's review this as part of the milestone 1 review to see how and if we would want to resolve this.

Reply:
@jjgrainger Ah that makes sense. In that case, I agree, let's not use `__()`.